src/JestExt.ts

```diff
@@ -459,7 +460,8 @@ export class JestExt {
   }
 
   private updateWithData(data: JestTotalResults) {
-    const normalizedData = resultsWithLowerCaseWindowsDriveLetters(data)
+    const noAnsiData = resultsWithoutAnsiCharacters(data)

```

Comment:
a bit nitpicking here, technically we are just removing the ANSI escape sequence, not all ANSI characters... rename accordingly would make it less confusing.

Reply:
Good point. Fixed.